batime@martin.zurowietz.de/extension.js

```diff
@@ -6,14 +6,14 @@ class BaTimeExtension {
    constructor() {
       this.aggregateMenu = Panel.statusArea.quickSettings;
       this.originalIndicator = this.aggregateMenu._system;
-      this.aggregateMenu._system = this.customIndicator = new BaTime.imports.power.Indicator();
+      this.aggregateMenu._batteryTime = this.customIndicator = new BaTime.imports.power.Indicator();
```

Comment:
Is this change related to GNOME 44 support as well (i.e. the compatibility with the other extension) or is this unrelated to the GNOME version?

Reply:
It's to fix #14

Reply:
Ok thanks! Would you please push the fix to a separate PR if it's unrelated to GNOME 44 support?

Reply:
Feels like extra work for little gain, but sure I have nothing going on right now.

Reply:
Thanks! It's just that I'd like to test the extension in GNOME 44 first but I can release the fix right away.